src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbvariantcontent.directive.js

```diff
@@ -7,12 +7,11 @@
     var umbVariantContent = {
         templateUrl: 'views/components/content/umb-variant-content.html',
         bindings: {
-            content: "<",
+            content: "=",
             page: "<",
-            editor: "<",
+            editor: "=",
```

Comment:
Having 2 way binding goes against how components 'should' work. The standard approach to components is to have one way binding and then callbacks else we fall into the previous angular traps where you don't know what is actually happening. Any reason why we need this? https://docs.angularjs.org/guide/component#component-based-application-architecture

Reply:
Good input, let me have a look to see what we can do about that.

Reply:
Gone now, I will mark this feedback as resolved.